### ISSUE TITLE
ShowOnce() Header does show on every page with 2024.7

### ISSUE THREAD

**Issue by Zwergenpunk** [QuestPDF/QuestPDF#957]
**Describe the bug**
The header with showOnce() is shown on every page with new version 2024.7. With 2024.6.x everything works

**To Reproduce**
```
page.Header().Column(col =>
			{
				col.Item().ShowOnce().Component<FirstPageHeaderComponent>();
				col.Item().SkipOnce().Component<FollowingPageHeaderComponent>();
				}
			});
```

**Expected behavior**
ShowOnce shows only once :)

**Screenshots**
![image](https://github.com/user-attachments/assets/aa8c5e47-b3ce-49af-addc-1454903f62d6)

**Environment**
linux + windows

**Comment by eubelov** [comment]
Same issue

**Comment by MarcinZiabek** [comment]
Thank you for sharing this issue 😄 

The problem should be fixed in the latest `2024.7.1` version. Could you please give it a try?

**Comment by Zwergenpunk** [comment]
looks good
